### PR TITLE
OCPQE-25316 add auto release job for 4.13

### PR DESCRIPTION
**Pull request by rioliu-rh**
- add e2e test jobs
- add upgrade jobs for upgrade path 4.13-stable to 4.13 stable
- add upgrade jobs fro upgrade path 4.12-stable to 4.13 stable
- use typical installation profiles

### CODE REVIEW

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-automated-release-aws-ipi-f360

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-automated-release-gcp-ipi-f360

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-automated-release-azure-ipi-f360

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-automated-release-stable-4.13-upgrade-from-stable-4.12-gcp-ipi-f360

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-automated-release-stable-4.13-upgrade-from-stable-4.13-gcp-ipi-f360

**Comment by rioliu-rh** [comment]
issue found in e2e test jobs is identified as automation issue [OCPQE-25337](https://issues.redhat.com/browse/OCPQE-25337)

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-automated-release-aws-ipi-f360

**Comment by rioliu-rh** [comment]
@liangxia can you review again, presubmit job failure is fixed

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-automated-release-stable-4.13-upgrade-from-stable-4.12-azure-ipi-f360

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__automated-release-stable-4.13-upgrade-from-stable-4.12.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+base_images:
+  ansible:
+    name: "4.13"
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__automated-release-stable-4.13-upgrade-from-stable-4.12.yaml:3]: Most of the images are used for installation, so the version should align with the "upgrade from" version

Comment (rioliu-rh) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__automated-release-stable-4.13-upgrade-from-stable-4.12.yaml]: thanks submitted new change according to this

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-automated-release-stable-4.13-upgrade-from-stable-4.12-azure-upi-f360

**Pull request merged**